src/app/layout.tsx

```diff
@@ -1,35 +1,17 @@
-import type { Metadata } from "next";
-import localFont from "next/font/local";
-import "./globals.css";
+import React from 'react';
+import './globals.css';
 
-const geistSans = localFont({
```

Comment:
Why are we deleting these?

Reply:
actually i done another way its done like inside of layout font specified but i done in global css same thing   because font should be managed

Reply:
@kunaldevxxx  Please see the Nextjs convention for handling fonts. I think they support managing fonts even when they are within app sources and need not be explicitly global

https://nextjs.org/docs/pages/building-your-application/optimizing/fonts